go.mod

```diff
@@ -89,7 +89,7 @@ require (
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
 	github.com/rancher/eks-operator v1.0.9
-	github.com/rancher/gke-operator v1.1.1-rc7
+	github.com/rancher/gke-operator v1.1.1-rc10
```

Comment:
Doesn't this need to get updated here ? https://github.com/rancher/system-charts/blob/dev-v2.5/charts/rancher-gke-operator/1.0.400/values.yaml#L7

Reply:
We could for completeness but it won't technically affect the bug, the bug was when rancher calls BuildUpstreamClusterState

Reply:
I think we should keep 2.6 and 2.5 on same version since that is what QA is expecting. If there are chart changes we could re-consider, but even then I'd think we should do it.

Reply:
They are the same version https://github.com/rancher/rancher/pull/33809/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R104

I haven't made a change to feature charts for 2.6, are you asking me to update feature charts and system-charts?

Reply:
Oh, yea I think we should keep all 4 (both charts and both rancher lines) on same version of operator. Not split them up.

Reply:
https://github.com/rancher/charts/pull/1378 https://github.com/rancher/system-charts/pull/510